### PR TITLE
Solomon places 5

### PR DESCRIPTION
**Pull request by sologebre**

### CODE REVIEW

File: 1001-2000/LOC1264Adwa.xml [hunk 1]
```diff
@@ -54,14 +54,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>ʿĀdwā</placeName>
-               <placeName type="alt">ʿĀduwā</placeName>
+            <place type="town" sameAs="Q380096">
+               <placeName xml:lang="gez" xml:id="n1">ዓድዋ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">ʿĀdwā</placeName>
+               <placeName xml:lang="gez" xml:id="n2" type="alt">ዓዱዋ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n2">ʿĀduwā</placeName>
```
Comment (DariaElagina) [inline 1001-2000/LOC1264Adwa.xml]: type normalized

File: 1001-2000/LOC1617Awas.xml [hunk 2]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="river">
-               <placeName>Awaš</placeName>
+            <place type="river" sameAs="Q782845">
+               <placeName xml:lang="gez" xml:id="n1">አዋሽ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">ʾAwāš</placeName>
```
Comment (DariaElagina) [inline 1001-2000/LOC1617Awas.xml]: type normalized

File: 1001-2000/LOC1619Awasa.xml [hunk 3]
```diff
@@ -59,8 +59,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Awasa</placeName>
+            <place type="town" sameAs="Q755765">
+               <placeName xml:lang="gez" xml:id="n1">አዋሳ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">ʾAwāsā</placeName>
```
Comment (DariaElagina) [inline 1001-2000/LOC1619Awasa.xml]: type normalized

File: 1001-2000/LOC1721BaherD.xml [hunk 4]
```diff
@@ -55,13 +55,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Bāḥǝr Dār</placeName>
+            <place type="town" sameAs="Q464699">
+               <placeName xml:lang="gez" xml:id="n1">ባሕር፡ ዳር፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Bāḥǝr Dār</placeName>
```
Comment (DariaElagina) [inline 1001-2000/LOC1721BaherD.xml]: type normalized

File: 2001-3000/LOC2658DerreD.xml [hunk 5]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Dǝrre Dawa</placeName>
+            <place type="town" sameAs="Q193486">
+               <placeName xml:lang="gez" xml:id="n1">ድሬ፡ ዳዋ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Dǝrre Dāwā</placeName>
```
Comment (DariaElagina) [inline 2001-3000/LOC2658DerreD.xml]: type normalized

File: 2001-3000/LOC2683Dilla.xml [hunk 6]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Dilla</placeName>
+            <place type="town" sameAs="Q905423">
+               <placeName xml:lang="gez" xml:id="n1">ዲላ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Dillā</placeName>
```
Comment (DariaElagina) [inline 2001-3000/LOC2683Dilla.xml]: type normalized

File: 3001-4000/LOC3009Eteya.xml [hunk 7]
```diff
@@ -54,13 +54,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Ǝteya</placeName>
+            <place type="town" sameAs="Q3155920">
+               <placeName xml:lang="gez" xml:id="n1">እቴያ፡
+               </placeName>
+               <placeName xml:lang="gez" corresp="#n1">ʾƎteyā</placeName>
+
```
Comment (DariaElagina) [inline 3001-4000/LOC3009Eteya.xml]: type normalized

File: 3001-4000/LOC3525Gobba.xml [hunk 8]
```diff
@@ -54,14 +54,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Gobba</placeName>
-               <placeName type="alt">Goba</placeName>
+            <place type="town" sameAs="Q745871">
+               <placeName xml:lang="gez" xml:id="n1">ጎባ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Gobbā</placeName>
+
```
Comment (DariaElagina) [inline 3001-4000/LOC3525Gobba.xml]: type normalized

File: 3001-4000/LOC3786Harar.xml [hunk 9]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Harär</placeName>
+            <place type="town" sameAs="Q190184">
+               <placeName xml:lang="gez" xml:id="n1">ሐረር፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Ḥarar</placeName>
```
Comment (DariaElagina) [inline 3001-4000/LOC3786Harar.xml]: type normalized

File: 4001-5000/LOC4116Kombol.xml [hunk 10]
```diff
@@ -54,14 +54,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Kombolča</placeName>
-               <placeName type="alt">Kombälša</placeName>
+            <place type="town" sameAs="Q222420">
+               <placeName xml:lang="gez" xml:id="n1">ኮምቦልቻ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Kombolčā</placeName>
+               <placeName xml:lang="gez" xml:id="n2" type="alt">ኮምበልሻ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n2">Kombalšā</placeName>
```
Comment (DariaElagina) [inline 4001-5000/LOC4116Kombol.xml]: type normalized

File: 4001-5000/LOC4455Makana.xml [hunk 11]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Mäkanä Sälam</placeName>
+            <place type="town" sameAs="Q3304801">
+               <placeName xml:lang="gez" xml:id="n1">መካነ፡ ሰላም፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">Makāna Salām</placeName>
```
Comment (DariaElagina) [inline 4001-5000/LOC4455Makana.xml]: type normalized

File: 1001-2000/LOC1264Adwa.xml [hunk 12]
```diff
@@ -54,14 +54,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>ʿĀdwā</placeName>
-               <placeName type="alt">ʿĀduwā</placeName>
+            <place type="town" sameAs="Q380096">
+               <placeName xml:lang="gez" xml:id="n1">ዓድዋ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1">ʿĀdwā</placeName>
```
[the remark this answers is not in the document]
Comment (sologebre) [reply 1001-2000/LOC1264Adwa.xml]: I have adjusted all!

File: 1001-2000/LOC1240AgawAw.xml [hunk 13]
```diff
@@ -10,7 +10,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Agäw-Awi</title>
+            <title>ʾAgaw  ʾAwi</title>
```
[the remark this answers is not in the document]
Comment (sologebre) [reply 1001-2000/LOC1240AgawAw.xml]: yes

**Pull request merged**